features/explicit-resource-management.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2025-08-06
-name: ECMAScript Async Explicit Resource Management
-description: TODO
+name: Explicit resource management
+description: The `using` and `await using` declarations and the `dispose` and `asyncDispose` symbols control resources that are not automatically cleaned up by JavaScript. The `DisposableStack` and `AsyncDisposableStack` objects can group, dispose, and coordinate dependencies between multiple disposable resources.
```

Can we dodge a lot of this, with something like this? It skips overs the `DisposableStack` business, but I think it covers the actual application more clearly.

```suggestion
description: The `using` and `await using` declarations and the `dispose` and `asyncDispose` symbols manage the lifecycle of resources such as file handles, network connections, and streams that are not automatically cleaned up by JavaScript.
```

Or if we must keep the stack stuff:

```suggestion
description: The `using` and `await using` declarations and the `dispose` and `asyncDispose` symbols manage the lifecycle of resources such as file handles and streams. The `DisposableStack` and `AsyncDisposableStack` objects can group, dispose, and coordinate dependencies between multiple disposable resources.
```